ad5121.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _AD5121_H_
+#define _AD5121_H_
+
+#include <stdint.h>
+
+enum AD5121_ID {
+  AD5121_0 = 0,
```

Comment:
Shouldn't we have the words "gain" and "offset" in this enum? It would make the code easier to follow.